style.css

```diff
@@ -545,6 +688,57 @@ body {
     font-size: 2.3rem;
   }
 
+  .modal-info {
+    width: 80%;
+    line-height: 25px;
+    padding: 3rem;
+    height: 100%;
+    margin-top: 0;
+    margin-bottom: 12rem;
+  }
+
+  .modal-dialog {
+    width: 150rem;
+    height: 500rem;
+  }
+
+  .modal-top-content {
+    margin-bottom: 20rem;
+  }
```

Comment:
Great job @GordonOtieno 🤝

- [ ] [Optional] You may kindly fix the responsiveness of your desktop popup window.
![responsive](https://user-images.githubusercontent.com/63917725/180191561-0d97e40b-8d8c-4a9e-95f7-aca6f23a78a6.PNG)